backend/agents/analysis_agent.py

```
@@ -133,44 +133,50 @@ async def _analyze_web_data(self) -> int:
                 sentiment=Sentiment(analysis.sentiment),
-                relevance=analysis.relevance_score
+                relevance=analysis.relevance_score,
             )
-            
+
             self.db.add(result)
             self.db.commit()
-            
+
             logger.info(f"Analyzed item: {data['source_id']}")
             return result
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)